it/src/test/java/com/expediagroup/streamplatform/streamregistry/it/StreamTestStage.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (C) 2018-2021 Expedia, Inc.
```

Comment:
Are changes to this file still necessary?

Reply:
yeah because now, we have check if schema is not matching with existing schemaKey.